templates/docs/db/buffalo-integration.es.md

```diff
@@ -0,0 +1,59 @@
+<% seoDescription("¿Cómo usar Pop con Buffalo?") %>
+<% seoKeywords(["buffalo", "go", "golang", "pop", "buffalo", "integración"]) %>
+
+<%= h1("Integración de Buffalo") %>
+
+Buffalo proporciona un middleware Pop para facilitar el uso de la base de datos dentro de Buffalo: https://github.com/gobuffalo/buffalo-pop
+
+<%= title("El Middleware de Transacción Pop") %>
+
+### Configuración
+
+Este middleware está configurado para usted de manera predeterminada, si elige usar Pop al crear un nuevo proyecto.
+
+**actions/app.go**
+
+```go
+func App() *buffalo.App {
+	if app == nil {
+        // [...]
+
+        app.Use(poptx.PopTransaction(models.DB))
+
+        // [...]
+
+        app.GET("/", HomeHandler)
+    }
+
+    return app
+}
+```
+
+`poptx.PopTransaction(models.DB)` utiliza la conexión a la base de datos configurada para crear un nuevo middleware `PopTransaction`. Este middleware hace lo siguiente:
+
+* Registre la duración total gastada durante la solicitud de llamadas a la base de datos.
+* Ajustar ** cada solicitud HTTP ** en una transacción de base de datos.
```

Comment:
Lo mismo acá, no deberían haber espacios entre la palabra y los asteriscos

Reply:
Fixed all the of text bolding